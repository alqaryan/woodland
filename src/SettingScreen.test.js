import renderer from 'react-test-renderer';
import React from 'react';

const SettingsScreen = require('./SettingsScreen.js');
describe('SettingsScreen', function() {
    it('should be exist', function() {
      expect('SettingsScreen').toBeDefined();
    });

    it('should be a function', function(){
      expect(typeof SettingsScreen).toBe("function");
    })

    it('renders correctly', () => {
      const tree = renderer.create(<SettingsScreen/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

  });