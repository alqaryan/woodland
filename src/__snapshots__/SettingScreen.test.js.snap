// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsScreen renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "marginBottom": 48,
        "marginHorizontal": 30,
      }
    }
  >
    <View
      style={
        Object {
          "marginTop": 32,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8A8F9E",
            "fontSize": 10,
            "textTransform": "uppercase",
          }
        }
      >
        Name
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        rejectResponderTermination={true}
        style={
          Object {
            "borderBottomColor": "#8A8F9E",
            "borderBottomWidth": 0.5,
            "color": "#8A8F9E",
            "fontSize": 15,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      style={
        Object {
          "marginTop": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8A8F9E",
            "fontSize": 10,
            "textTransform": "uppercase",
          }
        }
      >
        email
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        rejectResponderTermination={true}
        style={
          Object {
            "borderBottomColor": "#8A8F9E",
            "borderBottomWidth": 0.5,
            "color": "#8A8F9E",
            "fontSize": 15,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <View
      style={
        Object {
          "marginTop": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#8A8F9E",
            "fontSize": 10,
            "textTransform": "uppercase",
          }
        }
      >
        Password
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "borderBottomColor": "#8A8F9E",
            "borderBottomWidth": 0.5,
            "color": "#8A8F9E",
            "fontSize": 15,
            "height": 40,
          }
        }
        underlineColorAndroid="transparent"
        value=""
      />
    </View>
    <RNGestureHandlerButton
      collapsable={false}
      onGestureEvent={[Function]}
      onGestureHandlerEvent={[Function]}
      onGestureHandlerStateChange={[Function]}
      onHandlerStateChange={[Function]}
      rippleColor={0}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          undefined,
        ]
      }
    >
      <View
        accessible={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "green",
            "borderRadius": 4,
            "height": 52,
            "justifyContent": "center",
            "marginHorizontal": 30,
            "marginTop": 20,
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFF",
            }
          }
        >
          Update Email
        </Text>
      </View>
    </RNGestureHandlerButton>
    <RNGestureHandlerButton
      collapsable={false}
      onGestureEvent={[Function]}
      onGestureHandlerEvent={[Function]}
      onGestureHandlerStateChange={[Function]}
      onHandlerStateChange={[Function]}
      rippleColor={0}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          undefined,
        ]
      }
    >
      <View
        accessible={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "green",
            "borderRadius": 4,
            "height": 52,
            "justifyContent": "center",
            "marginHorizontal": 30,
            "marginTop": 20,
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFF",
            }
          }
        >
          Update Password
        </Text>
      </View>
    </RNGestureHandlerButton>
    <RNGestureHandlerButton
      collapsable={false}
      onGestureEvent={[Function]}
      onGestureHandlerEvent={[Function]}
      onGestureHandlerStateChange={[Function]}
      onHandlerStateChange={[Function]}
      rippleColor={0}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          undefined,
        ]
      }
    >
      <View
        accessible={true}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "green",
            "borderRadius": 4,
            "height": 52,
            "justifyContent": "center",
            "marginHorizontal": 30,
            "marginTop": 20,
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFF",
            }
          }
        >
          Update Name
        </Text>
      </View>
    </RNGestureHandlerButton>
  </View>
</View>
`;
